streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreChangelogReaderTest.java

```diff
@@ -642,11 +645,12 @@ public void shouldRestoreFromBeginningAndCheckCompletion() {
 
     @Test
     public void shouldCheckCompletionIfPositionLargerThanEndOffset() {
+        setupActiveStateManager();
```

Comment:
I suppose the point of not doing this in @Before is to avoid unnecessary-mocking errors?

Reply:
Yeah, the reason for not doing it in @before is that since not all tests exercise the methods in the stubbing Mockito complains that there are unused stubs. This way we only stub in the tests where the code is exercised!